package.json

```diff
@@ -7,9 +7,8 @@
     "build:clean": "rimraf dist && mkdirp dist && cp res/.htaccess res/zee-worker.js res/analytics.js dist/",
     "build:quiet": "yarn build:clean && env NODE_ENV=development webpack",
     "build": "yarn build:quiet --progress",
-    "build-prod:quiet": "yarn build:clean && env NODE_ENV=production webpack -p",
```

Comment:
`-p` enables the default uglify (I think this is short for `production`), but we explicitely use the newer plugin now.